### PR TITLE
PREAPPS-4183 : adding login page changes for zimbra9

### PR DESCRIPTION
**Pull request by auminpatel**
Adding zimbra9 login page changes
Adding css changes and adding new template for the two - factor authentication
Changing background image and colour and adding footer message
Updated the client info message for tooltip
Changing password-recovery template to match modern ui and change in css.
Adding color in skin.properties for branding and moving background image and color to a variable

### CODE REVIEW

File: WebRoot/css/login.css [hunk 1]
```diff
@@ -43,6 +43,46 @@ FORM {
 	padding:0px;
 }
 
+.signIn {
+	font-size: 20px;
+	text-align: left;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: We can remove it as it inherited from `#zLoginForm`

File: WebRoot/css/login.css [hunk 2]
```diff
@@ -43,6 +43,46 @@ FORM {
...
+	background: #eee;
+}
+
+.versionBlock label {
+	display: block;
+	padding-bottom: 3px;
+}
+.LoginScreen h1.logo{
+	margin: 0 30px;
+	overflow: hidden;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: do we need this?

File: WebRoot/css/login.css [hunk 3]
```diff
@@ -75,6 +115,33 @@ FORM {
...
  */
 .LoginScreen FORM 	{ text-align:center; 	}  /* fixes IE centering of form */
+
+#zLoginForm{
+	text-align: left;
+}
+
+form#zLoginForm .form {
+	color: #1e282ec2;
+	position: inherit;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: do we need this?

File: WebRoot/css/login.css [hunk 4]
```diff
@@ -107,23 +237,56 @@ FORM {
...
 .LoginScreen #ZLoginWhatsThisAnchor {
-	color:@LoginLabelColor@;
-	font-size:.9em;
-	margin-left:5px;
+	color: #1e282ec2;
+	font-size: .9em;
+	margin-left: 8px;
+	position: absolute;
+	left: 100%;
+	TOP: 8px;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: UPPERCASE ?

File: WebRoot/css/login.css [hunk 5]
```diff
@@ -107,23 +237,56 @@ FORM {
...
+	margin: 0em 1em 0 -2em;
+	position: absolute;
+	top: 25px;
+	z-index: 3;
+	height: 350px;
+	font-size: 14px;
+	padding: 24px 32px;
+	border-radius: 5px;
+	border-style: none;
+	box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);
```
Comment (lavegupta) [inline WebRoot/css/login.css]: need to re think about this.

File: WebRoot/css/login.css [hunk 6]
```diff
@@ -134,6 +297,10 @@ FORM {
 	@cssShadow@:0px 0px 2px black;
 }
 
+.loginTitle{
+	padding: 0px 40px;
+	text-align: left;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: We can remove it as it inherited from #zLoginForm

File: WebRoot/skins/_base/base3/skin.css [hunk 7]
```diff
@@ -229,7 +229,9 @@ BODY									{	margin:0px; 	}
 	overflow:hidden;
 	@FontFamily-default@
 	@FontSize-normal@
-	@LoginScreen@
+	background-color: #0088c1;
+	background-image: url(../../../img/new-back-ground-image.png);
+	background-size: cover;
```
Comment (lavegupta) [inline WebRoot/skins/_base/base3/skin.css]: background: #0088c1 url(../../../img/new-back-ground-image.png) 0 0/cover no-repeat;

File: WebRoot/css/login.css (hunk 5 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: how about this 

position: absolute;
    top: 23px;
    z-index: 3;
    height: 350px;
    font-size: 14px;
    left: 15px;
    right: 15px;
    padding: 24px 32px;
    border: 5px none;
    box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);

Comment (auminpatel) [reply WebRoot/css/login.css]: how about this 

```
position: absolute;
    top: 23px;
    z-index: 3;
    height: 350px;
    font-size: 14px;
    left: 15px;
    right: 15px;
    padding: 24px 32px;
    border: 5px none;
    box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);
```

File: WebRoot/skins/_base/base3/skin.css (hunk 7 above)
Comment (auminpatel) [reply WebRoot/skins/_base/base3/skin.css]: Doen

File: WebRoot/css/login.css (hunk 6 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: Done

File: WebRoot/css/login.css (hunk 4 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: Done

File: WebRoot/css/login.css (hunk 3 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: Done

File: WebRoot/css/login.css (hunk 2 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: Nope , have removed it

File: WebRoot/css/login.css (hunk 1 above)
Comment (auminpatel) [reply WebRoot/css/login.css]: Done

File: WebRoot/css/login.css [hunk 8]
```diff
@@ -107,24 +234,59 @@ FORM {
...
-	z-index:30; 
+	position: absolute;
+	top: 23px;
+	z-index: 3;
+	height: 350px;
+	font-size: 14px;
+	left: 15px;
+	right: 15px;
+	padding: 24px 32px;
+	border: 5px;
```
Comment (lavegupta) [inline WebRoot/css/login.css]: border: none;

Comment (auminpatel) [reply WebRoot/css/login.css]: Done

**Pull request merged**